indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+          <a class="usa-summary-box__link" href="#description">Description</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#tags--keywords">Tags</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#created-date">Created Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#changed-date">Changed Date</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#crawling-metadata">Crawling Metadata</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#thumbnail-image">Thumbnail Image</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#audience">Audience</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#content-type">Content Type</a> 
+        </li>
+        <li>
+          <a class="usa-summary-box__link" href="#searchgov-custom-fields">Search.gov Custom Fields</a> 
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+### Fields by Type
+
+|[Standard Fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)|[OpenGraph Fields](https://ogp.me/)|[DublinCore Fields](https://www.dublincore.org/)|
+|:----|:----|:----|:----|
+|[`<main>`](#main-page-content)|[`og:title`](#title)|[`dcterms.keywords`](#tags--keywords)|
+|[`<title>`](#title)|[`og:description`](#description)|[`dc.subject`](#tags--keywords)|
+|[`description`](#description)|[`article:section`](#tags--keywords)|[`dcterms.subject`](#tags--keywords)|
+|[`keywords`](#tags--keywords)|[`article:tag`](#tags--keywords)|[`dcterms.audience`](#audience)|
+|[`robots`](#crawling-metadata)|[`article:published_time`](#created-date)|[`dc.type`](#content-type)|
+||[`article:modified_time`](#changed-date)|[`dcterms.type`](#content-type)|
+||[`og:image`](#thumbnail-image)|[`dc.date`](#created-date)|
+|||[`dc.date.created`](#created-date)|
+|||[`dcterms.created`](#created-date)|
+
+
+### Main Page Content
+We look for the following field:
+```
+<main>
+```
+- **Detail:** Allows the search engine to target the actual content of the page and avoid headers, sidebars and other page content not useful to search. [Read more about the <main> element here]({{ site.baseurl }}/indexing/how-search-engines-index-content-better-discoverability.html#main-element)
+- **Used In:** Query matching, term frequency scoring
+
+### Title
+
+We look for the following fields:
+```
+<title>
+<meta property="og:title" content="Example" />
+```
+- **Detail:** Unique title of the page. If you want to include the agency or section name, place that after the actual page title. We use the `og:title` property as the result title if it appears to be more substantive than the `<title>` tag. Note, Open Graph elements are used to display previews of your content in FaceBook and some other social media platforms.
+- **Data Gathering Logic:** We prefer whichever title field is longer between the two.
```

Comment:
Seeing this in preview made me realize it might be colloquial enough to be confusing. how about "We collect both fields, and use whichever is longer in search results."

Reply:
fixed (Added this to the data gathering logic section)